spec/spec_helper.rb

```diff
@@ -8,6 +8,8 @@
   config.cassette_library_dir = File.join(ManageIQ::Providers::Openstack::Engine.root, 'spec/vcr_cassettes')
 end
 
+NotificationType.seed
```

Comment:
This should be moved to https://github.com/ManageIQ/manageiq/blob/master/lib/evm_database.rb in following PR, no merge blocker.